cicd/3-app/javabuilder/integration-test-buildspec.yml

```diff
@@ -0,0 +1,28 @@
+version: 0.2
+phases:
+  install:
+    runtime-versions:
+      nodejs: 14
+  pre_build:
+    commands:
+      - node -v
+      - npm -v
+      - cd ./integration-tests
+      - npm install
+  build:
+    commands:
+      - set -e
+
+      - # TODO: Make the ARN and/or secret names configurable too?
+      - private_key=$(aws secretsmanager get-secret-value --secret-id arn:aws:secretsmanager:us-east-1:475661607190:secret:development/cdo/javabuilder_integration_tests_key-b4BQG2)
+      - password=$(aws secretsmanager get-secret-value --secret-id arn:aws:secretsmanager:us-east-1:475661607190:secret:development/cdo/javabuilder_integration_tests_key_password-sWpRnW)
+
+      - # TODO: Make these configurable
+      - sub_domain=javabuilder-test
+      - base_domain=code.org
+
+      - JAVABUILDER_PRIVATE_KEY=$private_key \
+      - JAVABUILDER_PASSWORD=$password \
+      - JAVABUILDER_HTTP_URL=https://"$sub_domain"-http."$base_domain"/seedsources/sources.json \
+      - JAVABUILDER_WEBSOCKET_URL=wss://"$sub_domain"."$base_domain" \
```

Comment:
Ah since our testing setup on main got a few commits ahead of this, I think we'll need to add these two lines here ([source](https://github.com/code-dot-org/javabuilder/blob/d34ad98dab1d38db556b3a0c7aaffe1b869f7182/integration-tests/run-integration-tests.sh#L22)):
```
JAVABUILDER_BASE_DOMAIN=$base_domain \
JAVABUILDER_SUB_DOMAIN=$sub_domain \
```

Reply:
Thanks, I'll check how that's set up and make sure these align.

Reply:
Do you need all of the old env variables now? Can the two URL's be inferred by the base domain and subdomain?

Reply:
They can, but we'd need to update our test runner to do that because it currently expects both the domain/subdomain and URL variables.